docs/developers/celestia-app-vesting.md

```diff
@@ -0,0 +1,404 @@
+---
+sidebar_label: How to create a vesting account with Celestia App
+description: Learn how to generate a vesting account using Celestia App.
+---
+
+# How to create a vesting account with Celestia App
+<!-- markdownlint-disable MD013 -->
+
+In this guide, we will learn how to create a vesting account using
+celestia-appd for both a local devnet and on Mocha testnet.
+
+## Local devnet
+
+First, download and [install Celestia App](../../nodes/celestia-app),
+selecting the
+[network and corresponding version](../../nodes/participate)
+that you would like to use.
+
+### Setting up the local devnet
+
+#### Run the devnet
+
+Next, change into the `$HOME/celestia-app` directory and
+run the single-node-devnet script.
+
+```bash
+cd $HOME/celestia-app
+./scripts/build-run-single-node.sh
+```
+
+#### Save the home directory path
+
+At the top of the output, you will see a path to the "Home directory",
+find yours from the output (**it will be unique every time**):
+
+```bash
+./scripts/build-run-single-node.sh
+Home directory: /var/folders/_8/ljj6hspn0kn09qf9fy8kdyh40000gn/T/celestia_app_XXXXXXXXXXXXX.XV92a3qx
+--> Updating go.mod
+```
+
+And set the location as the `NODE_STORE` variable. We will use this
+for the remainder of the devnet section.
+
+```bash
+export NODE_STORE=/var/folders/_8/ljj6hspn0kn09qf9fy8kdyh40000gn/T/celestia_app_XXXXXXXXXXXXX.XV92a3qx
+```
+
+:::tip note
+This does not replace the `celestia-appd` binary that was
+installed with `celestia-appd`, but builds and runs one in the `/build`
```

Comment:
[nit] to clarify that the build directory referenced is local to the celestia-app repo and not some global `/build` directory

```suggestion
installed with `celestia-appd`, but builds and runs one in the `./build`
```

Reply:
added full path in this line for clarity in d1e148b5563e4b7e04a11f9e2d6c5917bab7ace6